config/api.yml

```diff
@@ -477,10 +477,21 @@
       :post:
       - :name: query
         :identifier: cloud_volume_show_list
+      - :name: delete
+        :identifier: cloud_volume_delete
+      :delete:
```

Comment:
@abellotti I have removed the DELETE method on the collection and the related specs (6ca849f). Please review. Thanks.